### ISSUE TITLE
Blog announcement

### ISSUE THREAD

**Issue by schne137** [nextcloud/polls#2874]
### Your Message to the world

Nextcloud would like to write a blog about Nextcloud Polls. It would feature its background story, features & improvements, contributers and future goals. If you are interested in collaborating, please comment and our team will reach out! :) Thanks!

**Comment by dartcafe** [comment]
Yes sure.

**Comment by dartcafe** [comment]
@v1r0x

**Comment by v1r0x** [comment]
Yeah, if this is really offically realted to nextcloud (can't find anything on @schne137 profile)

**Comment by schne137** [comment]
Thanks for your responses! @dartcafe and @v1r0x you can email mikaela@nextcloud.com. I write the blogs for Nextcloud here: https://nextcloud.com/blog/ (if you click on one, you can see my name there).

**Comment by nickvergessen** [comment]
I confirm the identity of @schne137 to be part of Nextcloud's marketing team

**Comment by v1r0x** [comment]
Thank you @schne137 and @nickvergessen !
